Accepted-RFCs/RFC-014-product-vision-templates-1.0.md

```diff
@@ -0,0 +1,196 @@
+
+The template leads would brainstorm, collaborate, and sign off on what is 1.0. No need for a separate RFC for that, though the conversation would be open to the entire project, not closed.
+
+Template contributors should get to weigh in and help craft as well, though not to the point where the scope of 1.0 spirals into something that isn't managable. To that point: it's likely that elements may be deemed to be "post 1.0" or "let's get part of that in 1.0, and put the rest on our post-1.0 roadmap."
+
+### "Why is what's 'core' a follow-up conversation?"
+
+This sort of working is a big shift in our working models, so before we get into specifics, we should come to an accord on this overall.
+
+This includes discussions about what framework the product takes, such as the "template packs" discussions.
+
+### Ideal timeline
+
+Given our tethering to the Write the Docs calendar, here's a proposal:
+
+ - We have a "0.9 release candidate" ready and out 2 weeks prior to WTD Portland
+	 - That requires some of the content strategy work done regarding users being able to navigate our site
+	 - We'll need to provide easy and clear ways for people to submit feedback to us
+ - We have a 1.0 release ready and out 2 weeks prior to WTD Prague
+
+ That said, if we can't hit this timeline, but the rest of this RFC looks legit to the steering committee, this element is deemed as flexible. (Similar to deciding what is/isn't 1.0.) However, the author strongly wants to assert a timeframe, as it's more beneficial to work toward a timeframe and fall short than it is to work without a timeframe.
+
+### "Wait, a 0.9 release candidate?"
+
+Yep. If we can have something that feels mostly complete, from a 1.0 perspective, but could either use testing or some more work, we can make that our 0.9. And if we have that 0.9 ready shortly before WTD Portland, we can present that at Writing Day and in Unconferences to:
+
+* Demonstrate concrete progress with a sense of "full release upcoming" we can share with the wider community
+* Better reach devs looking for docs help, by having a platform to speak both to devs and to writer friends such devs may have
+
+### Having a product leader role
+
+Part of what would help us forge a path toward a 1.0 is having someone keeping an eye of what as a group we're working on, and how those efforts can intersect with the product vision… or force the vision to adapt to reality. That would be a **product leader**.
+
+Ryan is proposing (in third person) of taking on that role, at least until we get to a 1.0 release. He doesn't have an attachment to any given template working group, so he's hoping that having a less biased viewpoint could help in offering product direction. (If anyone else wants to be a part of this role, that's welcomed! Though we'd like the tempalte leads to stick to production, and not split their focus.) 
```

Comment:
Praise: Yes, this leader role is definitely needed for a successful project. It requires someone who will prod, chase, mentor, but also be able to firmly but nicely say "no, you are not ready yet". I'm confident that Ryan would be very good at this.